src/Sheet.php

```diff
@@ -375,7 +374,7 @@ public function toArray($import, int $startRow = null, $nullValue = null, $calcu
      *
      * @return Collection
      */
-    public function toCollection($import, int $startRow = null, $nullValue = null, $calculateFormulas = false, $formatData = false): Collection
```

Comment:
I wouldn't want to change the default, that would be a breaking change for many users

Reply:
Please create a concern WithFormatData for this